### ISSUE TITLE
Issues with ubuntu 20.04 packages

### ISSUE THREAD

**Issue by gardar** [sni/mod_gearman#157]
I'm having some issues with the Ubuntu 20.04 packages for mod-gearman-worker and mod-gearman-module on a fresh install of v3.3.3, it seems that the config examples are missing from the package.

```
cp: cannot stat '/usr/share/doc/mod-gearman-worker/examples/worker.conf.gz': No such file or directory
dpkg: error processing package mod-gearman-worker (--configure):
 installed mod-gearman-worker package post-installation script subprocess returned error exit status 1
Setting up mod-gearman-module (3.3.3) ...
cp: cannot stat '/usr/share/doc/mod-gearman-module/examples/module.conf.gz': No such file or directory
dpkg: error processing package mod-gearman-module (--configure):
 installed mod-gearman-module package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 mod-gearman-worker
 mod-gearman-module
```

**Comment by snarf6** [comment]
Hola,

Seems to be related to #156 . Instead of reopening it, i'll put here what i found about it 

I've faced the same issue, and it was not related to `naemon-core` dep. 

I'm working on some docker integration (for specific use at work), and i think there's other dep issue. I've tried with `testing` branch

Using `debian:buster-slim` docker image, the install fail (for both worker and module), but it works with `debian:buster` out of the box. 

Something is assuming that a tool|package is installed in a base system, but, it's not. 

The game will be to hunt this dep
